docs/webhooks.rst

```diff
@@ -71,8 +82,8 @@ Using the generic API integration
 
 For repositories that are not hosted with a supported provider, we also offer a
 generic API endpoint for triggering project builds. Similar to webhook
-integrations, this integration has a specific URL, found on the
-:ref:`integration detail page <webhooks:Webhook Integrations>`.
+integrations, this integration has a specific URL, found on the project's
+**Integrations** Admin dashboard page on readthedocs.org.
```

Comment:
Also here, probably the users already know that the project is in readthedocs.org, right?

Reply:
Yeah, I'm revising in the next patch, see what you think... I still find it a bit disorienting if I'm not in that Dashboard very often.

Reply:
I think it is great now, we can change the wording here too. Thanks for the contribution btw ;)